src/components/images-and-icons/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { fileAbsolute } from 'paths.macro'
+import { getStoryTitle } from '../../../storybook/get-story-title'
+import { Avatar } from './Avatar'
+
+export default {
+  title: getStoryTitle(fileAbsolute),
+  component: Avatar,
+}
+
+export const Basic: React.FC = () => (
+  <Avatar
+    source={{ uri: 'https://cdn.pixabay.com/photo/2015/10/05/17/09/minion-972908_960_720.jpg' }}
```

Comment:
Pictures of Minions will not be tolerated in the codebase, please replace ASAP with [cute puppies](https://lmgtfy.com/?q=cute+puppy). 🐶

Reply:
Replaced with a cute puppy 

![Cute puppy](https://media.giphy.com/media/FOL5mK0tXUmXe/giphy.gif)